docker/compose.devcontainer.yaml

```diff
@@ -35,14 +35,14 @@ services:
       disable: true
     # https://docs.docker.com/reference/compose-file/services/#ports
     ports:
-      - host_ip: ${API_MAPPED_HOST_IP:?error}
-        name: api
-        published: ${API_MAPPED_PORT:?error}
-        target: ${API_PORT}
       - host_ip: ${API_DEBUGGER_MAPPED_HOST_IP:?error}
         name: api_debugger
         published: ${API_DEBUGGER_MAPPED_PORT:?error}
         target: ${API_DEBUGGER_PORT}
+      - host_ip: 0.0.0.0
```

Comment:
You have removed the possibility of someone restoring the old functionality with a variable. 

Change the value in the .env.devcontainer file instead